.github/workflows/rust.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       if: ${{ github.ref != 'refs/heads/main' }}
     - name: Report cargo version
       run: cargo --version
+    - name: Install coreutils
+      run: brew install coreutils
+      if: matrix.os == 'macos-12'
```

Comment:
I think this should probably go into install_builder_prerequisites.sh?  Part of the purpose of that is so that we're testing in CI exactly what developers following the usual build/test instructions would do.

Reply:
Hmm. I guess my primary hesitation is that on macos, this isn't the only way to ensure that the `sha256sum` command is installed, and running this command in `install_builder_prerequisites` could cause some real problems in some user environments because of how much homebrew does behind the scenes. What if a user prefers to manage their dependencies with `nix`, `asdf-vm`, or the like? How can we be sure they even have `brew` installed?

Reply:
Ah, I just saw that we are expecting users to have `brew` installed on macos environments:

```
levon@workstation-1:~/workspace/omicron$ rg brew
tools/install_builder_prerequisites.sh
153:    confirm "Install (or update) [${packages[*]}]?" && brew install "${packages[@]}"
```

Since that's the case, I can try moving this there.